src/core/transform.js

```diff
@@ -225,10 +225,13 @@ p5.prototype.scale = function() {
   for(var i = 0; i < args.length; i++) {
     args[i] = arguments[i];
   }
-  if(args[0] instanceof p5.Vector){
-    x = args[0].x;
-    y = args[0].y;
-    z = args[0].z;
+  // Only check for Vector argument type if Vector is available
+  if (typeof p5.Vector !== 'undefined') {
+    if(args[0] instanceof p5.Vector){
```

Comment:
let's put this together into one if:
`if (typeof p5.Vector !== 'undefined' && args[0] instanceof p5.Vector) {`